src/pages/cli/usage/mock.mdx

```diff
@@ -56,10 +58,23 @@ The environment variables that are listed below in [Function mock environment va
 
 In addition, you can use a `.env` file within the function directory (ie. `<project root>/amplify/backend/function/<function name>/.env`) to override any environment variables for local mocking.
 
+### Mocking the searchable models
+If you use `@searchable` directives on `@model` types in your GraphQL schema, then you can test the search GraphQL queries that are generated for you locally via `amplify mock` or `amplify mock api`.
+To learn more about the search queries, refer [this guide](/cli/graphql/search-and-result-aggregations/).
```

Comment:
Do you think we need an example GraphQL search query here? Different types of search queries are present in the guide I linked here. Would that suffice?